torch/lib/TH/generic/THTensorMath.c

```diff
@@ -1927,6 +1927,18 @@ void THTensor_(zeros)(THTensor *r_, THLongStorage *size)
   THTensor_(zero)(r_);
 }
 
+void THTensor_(zeros_like)(THTensor *r_, THTensor *input)
```

Comment:
nit: in TH/THC, almost all functions are camelCase. Should we enforce this here for consistency? (and let snake_case in the python interface)